crowbar_framework/lib/crowbar/proposal_methods.rb

```diff
@@ -0,0 +1,145 @@
+module Crowbar
+  module ProposalMethods
```

Comment:
Just so I understand: is this temporary until we get rid of ProposalObject, or do we plan to keep the public methods split forever?